### ISSUE TITLE
pf4j plugins with Gradle 5.0+

### ISSUE THREAD

**Issue by meydominic** [pf4j/pf4j#285]
When you upgrade your gradle in your plugin projects to 5.0 and newer you have to add the following line to build gradle dependencies:

```Gradle
annotationProcessor group: 'org.pf4j', name: 'pf4j', version: '2.6.0'`
```

So that you have:
```Gradle
compileOnly group: 'org.pf4j', name: 'pf4j', version: '2.6.0'
annotationProcessor group: 'org.pf4j', name: 'pf4j', version: '2.6.0'
```

If not your extensions won't be found.

**Comment by decebals** [comment]
Thanks that you share with us this information. I my projects I use Maven but probably they are many people that use Gradle.

**Comment by MacTrophy** [comment]
This should be documented in the demo_gradle files, at least in comments!

Thank you @meydominic

**Comment by decebals** [comment]
@claymccoy fixed the Gradle demo with #349. I see that line specified by you is [present](https://github.com/pf4j/pf4j/pull/349/files#diff-83e9ca6ad43154dcf909f8c8a363310fR7) in this PR.

**Comment by decebals** [comment]
I will close this issue for the moment.
